pkg/web/api/v1/transaction.go

```diff
@@ -54,7 +54,7 @@ type SecureEnvelope struct {
 	EnvelopeID          uuid.UUID    `json:"envelope_id"`
 	Direction           string       `json:"direction"`
 	Remote              string       `json:"remote,omitempty"`
-	ReplyTo             ulid.ULID    `json:"reply_to,omitempty"`
+	ReplyTo             *ulid.ULID   `json:"reply_to"`
```

Comment:
The pointer will hopefully ensure that this value is `null` in JSON rather than `0000000000`.